src/main.rs

```diff
@@ -52,7 +66,9 @@ async fn main() -> AppResult<()> {
             }
             Event::Reset(mode) => {
                 App::reset(mode.clone()).await?;
-                app = App::new(help.clone(), mode).await?;
+                app = App::new(help.clone(), config.clone(), mode).await?;
+                // Otherwise password dialog is not drawn until key pressed
+                tui.draw(&mut app)?;
```

Comment:
I don't think this is necessary cause the next instruction to run if `tui.draw` on the line 52.

Reply:
Might just have been a problem with my terminal.